### PR TITLE
ONI-95: hide add claim button

### PR DESCRIPTION
**Pull request by olewandowski1**
[ONI-95](https://openimis.atlassian.net/browse/ONI-95)

Change:
- Hide 'Create claim' button in top right corner.
- Disable 'View' button when user is not permitted.

[ONI-95]: https://openimis.atlassian.net/browse/ONI-95?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by wzglinieckisoldevelo** [comment]
LGTM

**Comment by sniedzielski** [comment]
Seems that sonar checks refers to the existing code therefore LGTM this PR.

**Review by sniedzielski — APPROVED** [review]

File: src/components/ClaimInsureeSummary.js [hunk 1]
```diff
@@ -41,8 +40,7 @@ const ClaimInsureeSummary = ({ insuree }) => {
   const { formatMessage, formatMessageWithValues, formatDateFromISO } = useTranslations(MODULE_NAME, modulesManager);
 
   const { claims, fetchingClaims, errorClaims, claimsPageInfo } = useSelector((store) => store.claim);
-
-  const onCreateNewClaim = () => historyPush(modulesManager, history, "claim.route.healthFacilities");
+  const { health_facility_id } = useSelector((store) => store.core.user.i_user);
```
Comment (sniedzielski) [inline src/components/ClaimInsureeSummary.js:43]: Please change to cammel case

**Review by sniedzielski — CHANGES_REQUESTED** [review]
Please refer to one comment, we can skip sonar warnings because those ones refers to old existing code, not to the new code.

File: src/components/ClaimInsureeSummary.js (hunk 1 above)
Comment (sniedzielski) [reply src/components/ClaimInsureeSummary.js:43]: I mean into 'healthFacilityId'

Comment (olewandowski1) [reply src/components/ClaimInsureeSummary.js]: Done

**Review by sniedzielski — APPROVED** [review]

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**